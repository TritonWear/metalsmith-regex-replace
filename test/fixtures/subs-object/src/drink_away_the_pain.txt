I used to be in love with this bitch named E&J
Don't fuck with her no more now I fuck with Tanqueray