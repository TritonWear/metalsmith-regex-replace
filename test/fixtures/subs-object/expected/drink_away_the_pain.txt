I used to be in love with this girl named E&J
Don't mess with her no more now I mess with Tanqueray